### PR TITLE
feat: enable search by bech32 address

### PR DESCRIPTION
**Pull request by luizstacio**
- Enable search for bech32 addresses on the search bar
- Enable linking using bech32 address ex.: [/address/fuel129wju39hs7ju0u4sm9enu9srvrv0af0m0epufej925e48gkrtjus9q9syg](https://fuellabs.github.io/block-explorer-v2/address/fuel129wju39hs7ju0u4sm9enu9srvrv0af0m0epufej925e48gkrtjus9q9syg?providerUrl=https%3A%2F%2Fnode-beta-1.fuel.network%2Fgraphql)
- Add feedback when the text input on the search box is not valid
   Screen Shot 2022-09-04 at 5 13 52 PM

### CODE REVIEW

File: src/pages/HomePage/index.tsx [hunk 1]
```diff
@@ -41,7 +42,7 @@ export default function HomePage() {
   });
   const navigate = useNavigate();
 
-  const isAllowedToSearch = searchText.length === 66;
+  const isAllowedToSearch = [66, 63].includes(searchText.length);
```
Comment (adlerjohn) [inline src/pages/HomePage/index.tsx:45]: Can you clarify why the change? It might be worth it to document here in the code too.

Comment (luizstacio) [reply src/pages/HomePage/index.tsx]: The idea was to validate by the length of the address, but it makes more sense to use the real validation of bech32 in this case. I made a change and added a comment and changed the user feedback.

**Review by QuinnLee — APPROVED** [review]
👍🏼 
There are some linting warnings, but I'll approve this for the test net launch
